### PR TITLE
[Feat/LIVE-7674]: add staking tracking for NEAR, ELROND

### PR DESCRIPTION
**Pull request by kallen-ledger**
### 📝 Description

Implementation of the following tag to track on the last CTA for completing a staking: 
- Validator chosen 
- Successful staking
- Source of staking flow

For the following programs where Ledger is validator: NEAR, ELROND

### ❓ Context

- **Impacted projects**: `LLM` 
- **Linked resource(s)**: `[LIVE-7674](https://ledgerhq.atlassian.net/browse/LIVE-7674)` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

N/A

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-7674]: https://ledgerhq.atlassian.net/browse/LIVE-7674?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/DelegationDrawer.tsx [hunk 1]
```diff
@@ -91,8 +91,8 @@ export default function DelegationDrawer({
         </ScrollView>
 
         <View style={[styles.row, styles.actionsRow]}>
-          {actions.map(props => (
-            <ActionButton {...props} />
+          {actions.map((props, i) => (
+            <ActionButton key={`actions-${i}`} {...props} />
```
Comment (kallen-ledger) [inline apps/ledger-live-mobile/src/components/DelegationDrawer.tsx:95]: Not related to PR, but an error noticed during development

File: apps/ledger-live-mobile/src/families/cosmos/DelegationFlow/04-ValidationSuccess.tsx [hunk 2]
```diff
@@ -34,17 +34,17 @@ export default function ValidationSuccess({ navigation, route }: Props) {
   const delegation = getTrackingDelegationType({
     type: route.params.result.type,
   });
-  const currency = getAccountCurrency(account);
+  const { ticker } = getAccountCurrency(account);
```
Comment (kallen-ledger) [inline apps/ledger-live-mobile/src/families/cosmos/DelegationFlow/04-ValidationSuccess.tsx:37]: Fix from last PR, should have used ticker

File: apps/ledger-live-mobile/src/families/solana/DelegationFlow/ValidationSuccess.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export default function ValidationSuccess({ navigation, route }: Props) {
   const delegation = getTrackingDelegationType({
     type: route.params.result.type,
   });
-  const currency = getAccountCurrency(account);
+  const { ticker } = getAccountCurrency(account);
```
Comment (kallen-ledger) [inline apps/ledger-live-mobile/src/families/solana/DelegationFlow/ValidationSuccess.tsx:35]: fix from last PR, should have used ticker

File: apps/ledger-live-mobile/src/families/near/StakingFlow/02-Summary.tsx [hunk 4]
```diff
@@ -164,12 +164,22 @@ export default function StakingSummary({ navigation, route }: Props) {
 
   const onContinue = useCallback(async () => {
     navigation.navigate(ScreenName.NearStakingSelectDevice, {
+      source: route.params.source,
+      validatorName: chosenValidator.validatorAddress,
```
Comment (sarneijim) [inline apps/ledger-live-mobile/src/families/near/StakingFlow/02-Summary.tsx:168]: Do you have a real validator name here? In LLD we have only a "recipient"???

**Review by sarneijim — APPROVED** [review]

File: apps/ledger-live-mobile/src/families/near/StakingFlow/02-Summary.tsx (hunk 4 above)
Comment (kallen-ledger) [reply apps/ledger-live-mobile/src/families/near/StakingFlow/02-Summary.tsx]: You are right. Updated 👍

File: apps/ledger-live-mobile/src/families/elrond/components/Flows/Delegate/components/ValidationSuccess/ValidationSuccess.tsx [hunk 5]
```diff
@@ -20,23 +21,34 @@ import styles from "./styles";
...
   const { colors } = useTheme();
   const { account } = useSelector(accountScreenSelector(route));
+  const { ticker } = getAccountCurrency(account);
 
   const parent = useMemo<StackNavigatorNavigation<BaseNavigatorStackParamList>>(
     () => navigation.getParent(),
     [navigation],
   );
 
+  const validator = route.params.validatorName;
```
[the remark this answers is not in the document]
Comment (kallen-ledger) [reply apps/ledger-live-mobile/src/families/elrond/components/Flows/Delegate/components/ValidationSuccess/ValidationSuccess.tsx]: Thank you, updated 👍

**Pull request merged**